linera-chain/src/manager/single.rs

```diff
@@ -53,10 +53,11 @@ impl SingleOwnerManager {
             ChainError::InvalidBlockProposal
         );
         if let Some(vote) = &self.pending {
-            let block = match &vote.value.inner() {
+            let block = match vote.value() {
                 CertificateValue::ConfirmedBlock { executed_block, .. } => &executed_block.block,
-                CertificateValue::ValidatedBlock { .. } => {
-                    return Err(ChainError::InvalidBlockProposal)
+                value => {
+                    let msg = format!("Unexpected value: {:?}", value);
```

Comment:
If the Debug string of the Value is large this could be a problem.

Reply:
It's unreachable anyway, because we only put `ConfirmedValue`s into `pending`.